packages/core/src/util/txInspector.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  Address,
+  CertificateType,
+  Ed25519KeyHash,
+  Lovelace,
+  PoolId,
+  StakeAddressCertificate,
+  StakeDelegationCertificate,
+  TokenMap,
+  TxAlonzo,
+  TxOut,
+  Value
+} from '../Cardano';
+
+const isMyAddress =
```

Comment:
```suggestion
const isAddressWithin =
```
The function isn't concerned with ownership, all it's doing is checking for presence in the given array.